### PR TITLE
scloud, scentre and srel

### PR DESCRIPTION
**Pull request by VorTechnix**
`//srel` seems stable... not 100% sure it is but I'm not sure else to do to test it.

### CODE REVIEW

File: worldeditadditions/utils/selector_helps.lua [hunk 1]
```diff
@@ -18,5 +18,21 @@ function worldeditadditions.axis_left(axis,sign)
 	else return true, "x", -sign end
 end
 
+--- Dehumanize Direction: translates up, down, left, right, front, into xyz based on player orientation.
+-- @param	name	string	The name of the player to return facing direction of.
+-- @param	dir	string	Relative direction to translate.
+-- @return	Returns axis name and sign multiplier.
+function worldeditadditions.dh_dir(name, dir)
```
Comment (VorTechnix) [inline worldeditadditions/utils/selector_helps.lua:25]: I know talked about changing this name... I guess now is the time to decide what to call this.

Comment (VorTechnix) [reply worldeditadditions/utils/selector_helps.lua]: Renamed to `xlate_dir`.

**Review by sbrl — COMMENTED** [review]
2 teensy suggestions, but awesome work!

File: .gitignore [hunk 2]
```diff
@@ -39,3 +39,5 @@ luac.out
 *.x86_64
 *.hex
 
+# Debug Tools (for now)
+*debug/
```
[the remark this answers is not in the document]
Comment (VorTechnix) [reply .gitignore]: changed to `worldeditadditions_debug/`

File: worldeditadditions_commands/commands/selectors/scentre.lua [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+local wea = worldeditadditions
+worldedit.register_command("scentre", {
+	params = "",
+	description = "Set WorldEdit region positions 1 and 2 to the centre of the current selection.",
+	privs = {worldedit=true},
+	require_pos = 2,
+	parse = function(params_text)
+		return true
+	end,
+	func = function(name)
```
[the remark this answers is not in the document]
Comment (VorTechnix) [reply worldeditadditions_commands/commands/selectors/scentre.lua:15]: Well I didn't need it so why do the extra typing to use it?

[the remark this answers is not in the document]
Comment (sbrl) [reply worldeditadditions_commands/commands/selectors/scentre.lua:15]: Ah I meant in general

**Pull request merged**